util/topgen/merge.py

```diff
@@ -1096,6 +1101,10 @@ def amend_interrupt(top: ConfigT,
             qual["intr_type"] = signal.intr_type
             qual["default_val"] = signal.default_val
             qual["incoming"] = False
+            if "plic" in ip:
+                qual["plic"] = ip["plic"]
+            else:
+                qual["plic"] = default_plic  # May still be None
```

Comment:
Following up on my suggested move to optional (the previous comment), I think this becomes
```python
            plic = ip.get("plic", default_plic)
            if plic is not None:
              qual["plic"] = plic
```